tests/io/test_io.py

```diff
@@ -9,9 +9,10 @@
 import pytest
 from packaging import version
 
-import pandera as pa
+import pandera
```

Comment:
any particular reason for this import change? seems like it incurs a lot of changes that don't seem relevant to the overall diff.

Reply:
Got an error while serializing to yaml about global pandera not available. I was perhaps too quick to "fix" that import error. I'll investigate the root cause.